stable/mongodb/values.yaml

```diff
@@ -313,7 +313,10 @@ metrics:
   ## Metrics exporter resource requests and limits
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   ##
-  # resources: {}
+  resources:
```

Comment:
You should revert this change. We do not suggest adding `resource.requests` in the default configuration.

Reply:
sure. I remove README.md default values

Reply:
perfect, Thanks!